gradle/profiles/profile-sapig-ob-mr1.gradle.kts

```diff
@@ -0,0 +1,67 @@
+/* ************************************************* */
+/* default profile                                   */
+/* ************************************************* */
+
+// Truststore configuration
+val truststorePath by extra("/com/forgerock/sapi/gateway/ob/uk/truststore.jks")
+val truststorePassword by extra("changeit")
+/**
+ * OB configuration: Used Create a Software Statement Assertion (SSA) using an API call for Dynamic Registration
+ * https://openbanking.atlassian.net/wiki/spaces/DZ/pages/1322979574/Open+Banking+Directory+Usage+-+eIDAS+release+Production+-+v1.9
+ */
+// OB directory organisation
+val obOrganisationId by extra("0015800001041REAAY")
+val obSoftwareId by extra("Y6NjA9TOn3aMm9GaPtLwkp")
```

Comment:
Update to Encore Software ID , can be found in values.yaml on line 12

Reply:
Ignore this, like below the software ID is tied to the automated testing certs not the mtls ones we create